src/notation/internal/notationactioncontroller.cpp

```diff
@@ -2040,7 +2040,7 @@ void NotationActionController::navigateToTextElement(MoveDirection direction, bo
         const Harmony* chordSymbol = editedChordSymbol();
 
         // otherwise, chord symbol will be deleted when navigating away from it
-        const bool canPlay = chordSymbol && !chordSymbol->plainText().empty();
+        const bool canPlay = chordSymbol && !chordSymbol->harmonyName().empty();
```

Comment:
I seem to recall there are two more instances of this code in this file. Might be good to fix this as well.